code/matt_r/python/lab_2b_palindrome.py

```diff
@@ -0,0 +1,46 @@
+
+# palindrome
+
+
+# # word = 'racecar'
+# # word = "newb"
+
+
+def check_palindrome(word):
+    word_list = list(word)
+    word_list_2 = list(word)
+    word_list_2.reverse()
+    word_list == word_list_2
```

Comment:
This line of code does not do anything, it's an evaluation, but the evaluation isn't used.